.github/workflows/test_configs.yml

```diff
@@ -0,0 +1,27 @@
+name: Test configs
+
+on:
+  push:
+    paths:
+      - .github/workflows/test_configs.yml
+      - Dockerfile
+      - vladdoster/.zshrc
```

Comment:
Might as well just normalize the configuration file to be `zshrc` or something. I'll leave that to your discretion. So the following suggestions work

Reply:
I did this in #11 :)